runtime/lua/vim/ui.lua

```diff
@@ -145,7 +145,8 @@ function M.confirm(msg, opts, on_choice)
   if not ok then
     on_choice(nil, nil)
   else
-    on_choice(choice_index, opts.choices[choice_index])
+    local epurate_choice = opts.choices[choice_index]:gsub('&', '')
```

Comment:
what does "epurate" mean? that word is new even to me...

Reply:
French word, it basically means purify, mybad I'm correcting this.

Reply:
"purify" is not great, either; people won't know what that means when reading the code. What was wrong with what you had before?

Reply:
The selected choice still contains the '&' used for shortcut so I had to remove it from the string.